app/src/main/java/com/example/dogadoption/retrofit/DogPictures.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.dogadoption.retrofit
+
+data class DogPictures(
+    val status: String,
+    val images: List<DogPicture>
+)
+
+data class DogPicture(
+    val url: String
+)
```

Comment:
This is wrong. Looking at the [documentation](https://dog.ceo/dog-api/documentation/breed) it should be

```kotlin
data class DogImages(
    val message: List<String>,
    val status: String
)
```